docs/migrate_molgenis/guide-migration.md

```diff
@@ -0,0 +1,70 @@
+# Migration
+In migration we define two steps:
+
+- Migrating operational requirements
+- Migration database schemes
+
+## Migrate from 8.0.x to 8.1.x
+
+### Migrate operational requirements
+There are a few services you need to migrate data from.
+
+- filestore 
+  - for 8.x /home/molgenis/
+- minio
+
+#### FileStore
+The filestore needs to be imported into Minio. The howto is coming soon.
+
+## Migrate from 7.x to 8.0.x
+There are a few major changes in the migration from 7.x to 8.x.
+- The postgres version was 9.6 and is now 11.0.1
+- The filestore is becoming minio from 8.1.x and forward
+
+### Migrate operational requirements
+There are a few services you need to migrate data from.
+
+- postgres
+  - for 7.x postgres 9.6
+  - for 8.x postgres 11.0.1
+- filestore 
+  - for 7.x /srv/molgenis/
+  - for 8.x /home/molgenis/
+  
+#### PostgreSQL
+Go to a terminal and dump your database using (as root-user):
+
+```bash
+cd /tmp/
+pg_dump molgenis > molgenis.sql
+```
+
+Load you database using on the new server (as root-user):
+
+```
+cd /tmp/
+psql molgenis < molgenis.sql
+```
+
+#### FileStore
+Go to a terminal and tar you files into an archive (as root-user):
+
+```
+cd /tmp/
+tar -pzcvf molgenis.tar.gz /home/molgenis
+```
+
+Restore the filestore on the new server (as root-user):
+
+```bash
+cd /tmp/
+tar -pxvzf molgenis.tar.gz
+```
+
+### Migrate database schemes
+The following EMX changes where made since the 7.x release:
+
+- MREF as labels are not allowed anymore. You should use template expressions. Please check: [template expressions](../user_documentation/import-data/ref-emx.md#template).
+  
```

Comment:
-  If users or groups were giving special permissions on the Settings Manager Plugin, these permissions should be set again for the replacement Settings Plugin. The plugin itself is replaced automatically.